_includes/home-page.html

```diff
@@ -148,9 +148,19 @@ <h2 class="sub-text" style="padding-top:1rem; font-size: 1rem; font-weight: 500;
 				<h1 class="created-by">Created By</h1>
 				<div class="company-image">
 					<a href="https://layer5.io/" target="_blank">
-						<img id="layer5-logo" src="./assets/images/company-logo/layer5-no-trim.svg" alt="Layer5 logo"
+						<img id="layer5-logo" alt="Layer5 logo"
 							loading="lazy" />
 					</a>
+					<script>
+						const layer5_logo = document.getElementById("layer5-logo");
```

Comment:
I think we already check this logic somewhere we don't have to describe everywhere this for the current theme. Use that logic . Check for meshery logo in header

Reply:
The line you've highlighted is not a logic. Its a variable assignment. What do you mean by "logic" here?

Reply:
The logic you used to check for the current theme whether its light or dark , is already there you need to reuse that javascript

Reply:
Hi @sudhanshutech thanks for the clarification. Fixed resuing the script.